templates/sheets/alchemical-sheet.html

```diff
@@ -1,19 +1,22 @@
 <form class="{{cssClass}}" autocomplete="off">
     <header class="sheet-header">
         <img class="profile-img" src="{{item.img}}" data-edit="img" title="{{item.name}}" />
-        <div class="header-fields">
-            <h1 class="charname"><input name="name" type="text" value="{{item.name}}" placeholder="Name" /></h1>
-            <div class="resource flex">
-                <div class="flex">
-                    <label>{{localize "WITCHER.Item.Quantity"}}</label>
-                    <input type="text" name="data.quantity" value="{{data.quantity}}" data-dtype="Number" />
-                    <label>{{localize "WITCHER.Item.Weight"}}</label>
-                    <input type="text" name="data.weight" value="{{data.weight}}" data-dtype="Number" />
-                    <label>{{localize "WITCHER.Item.Cost"}}</label>
-                    <input type="text" name="data.cost" value="{{data.cost}}" data-dtype="Number"/>
-                </div>
-            </div>
+        <h1 class="charname"><input name="name" type="text" value="{{item.name}}" placeholder="Name" /></h1>
```

Comment:
probably something I missed during developpemnt but class shouldn't be charname since it's an item name

Reply:
Same thing in other files

Reply:
Yeah noticed it too when I was coding, didn't want to change it since I didn't want to break anything but I could change it :P